packages/graphback/src/resolvers/outputResolvers/apollo.ts

```diff
@@ -0,0 +1,112 @@
+import { TargetResolverContext, TypeContext } from '../knex/targetResolverContext';
+
+export interface OutputResolver {
+  name: string
+  output: string
+}
+
+const imports = `import { GraphQLContext } from '../../context'`
+
+export const generateTypeResolvers = (context: TargetResolverContext, name: string): string => {
+  if(context.relations.length && context.subscriptions.length) {
+    return `${imports}
+  
+enum Subscriptions {
+  ${context.subscriptionTypes}
+}
+
+export const ${name}Resolvers = {
```

Comment:
This should be rather part of the template I think. The template should shape resolvers to the format that is required.

Reply:
Yes this would be for apollo-server

Reply:
When there will be multiple templates, I was thinking of having a key in the template and use respective template like apollo for now

Reply:
> This should be rather part of the template I think. The template should shape resolvers to the format that is required.

How would that work, in the cli ?

Reply:
Let's say we generate resolvers using Apollo model and this will not change. From Apollo Model we can move to graphql.js model easily but not vice versa. Then Graphq.js template will just transform it in code.